tests/conversion

```diff
@@ -247,13 +251,86 @@ lxc delete vm-tmp
 tar -xzf "${tmpdir}/vm-tmp.tar.gz" -C "${tmpdir}" "backup/virtual-machine.img"
 rm "${tmpdir}/vm-tmp.tar.gz"
 
+IMAGE_PATH="${tmpdir}/backup/virtual-machine.img"
+
 # Test VM migration using conversion mode. If server does not support
 # conversion API extension, lxd-migrate must fallback to migration
 # mode and successfully transfer the VM disk.
-conversion_vm vm-alpine-raw-btrfs btrfs "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-lvm lvm "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-zfs zfs "${tmpdir}/backup/virtual-machine.img" "no"
-conversion_vm vm-alpine-raw-dir dir "${tmpdir}/backup/virtual-machine.img" "no"
+conversion_vm vm-alpine-raw-btrfs btrfs "${IMAGE_PATH}" "no"
+conversion_vm vm-alpine-raw-lvm lvm "${IMAGE_PATH}" "no"
+conversion_vm vm-alpine-raw-zfs zfs "${IMAGE_PATH}" "no"
+conversion_vm vm-alpine-raw-dir dir "${IMAGE_PATH}" "no"
+
+# Test non-interactive VM imports.
+# Generate token for HTTPS remote server.
+token="$(lxc config trust add --quiet --name migrate)"
+if [ -z "${token}" ]; then
+    echo "Failed to generate LXD trust token!"
+    return 1
+fi
+
+# Target server test.
+lxd-migrate --non-interactive --instance-type vm --source "${IMAGE_PATH}" --instance-name vm-ni-unix-default \
+    --config security.secureboot=false
+testImportedInstance "${instName}"
```

Comment:
`s/${instName}/vm-ni-unix-default/` ?

Reply:
Agree, code would look nicer. I prefer having different names just because it is easier to find the relevant part in script if something goes wrong :)

Reply:
I see, thanks!